test/unit/test_assistant_v1.py

```diff
@@ -1,1576 +1,3973 @@
+            version='2019-02-28',
+        )
+        service.set_service_url(base_url)
+        output = service.delete_intent(**body)
+        return output
+
+    def construct_full_body(self):
+        body = dict()
+        body['workspace_id'] = "string1"
+        body['intent'] = "string1"
+        return body
+
+    def construct_required_body(self):
+        body = dict()
+        body['workspace_id'] = "string1"
+        body['intent'] = "string1"
+        return body
+
+
+# endregion
+##############################################################################
+# End of Service: Intents
+##############################################################################
+
+##############################################################################
+# Start of Service: Examples
+##############################################################################
+# region
+
+
+#-----------------------------------------------------------------------------
+# Test Class for list_examples
+#-----------------------------------------------------------------------------
+class TestListExamples():
+
+    #--------------------------------------------------------
+    # Test 1: Send fake data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_list_examples_response(self):
+        body = self.construct_full_body()
+        response = fake_response_ExampleCollection_json
+        send_request(self, body, response)
+        assert len(responses.calls) == 1
+
+    #--------------------------------------------------------
+    # Test 2: Send only required fake data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_list_examples_required_response(self):
+        # Check response with required params
+        body = self.construct_required_body()
+        response = fake_response_ExampleCollection_json
+        send_request(self, body, response)
+        assert len(responses.calls) == 1
+
+    #--------------------------------------------------------
+    # Test 3: Send empty data and check response
+    #--------------------------------------------------------
+    @responses.activate
+    def test_list_examples_empty(self):
+        check_empty_required_params(self, fake_response_ExampleCollection_json)
+        check_missing_required_params(self)
+        assert len(responses.calls) == 0
+
+    #-----------
+    #- Helpers -
+    #-----------
+    def make_url(self, body):
+        endpoint = '/v1/workspaces/{0}/intents/{1}/examples'.format(
+            body['workspace_id'], body['intent'])
+        url = '{0}{1}'.format(base_url, endpoint)
+        return url
+
+    def add_mock_response(self, url, response):
+        responses.add(responses.GET,
+                      url,
+                      body=json.dumps(response),
+                      status=200,
+                      content_type='applicatio
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'e' is assigned to but never used](https://app.codacy.com/manual/watson-developer-cloud/python-sdk/pullRequest?prid=4835642)